README.md

```diff
@@ -356,12 +338,20 @@ MemoryLocation can be used if you don't want router to touch the address bar at
 e.g.
 
 ```js
-var cherrytree = require('cherrytree')
-var MemoryLocation = require('cherrytree/lib/locations/memory')
-var router = cherrytree()
-routerlisten(new MemoryLocation())
+var router = cherrytree({
+  location: 'memory'
```

Comment:
I think this option would be better named as `locationProvider` or such - it's not the actual location.